net-misc/zerotier/zerotier-1.4.6.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic llvm systemd toolchain-funcs
+
+HOMEPAGE="https://www.zerotier.com/"
+DESCRIPTION="A software-based managed Ethernet switch for planet Earth"
+SRC_URI="https://github.com/zerotier/ZeroTierOne/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSL-1.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+IUSE="clang neon"
+
+S="${WORKDIR}/ZeroTierOne-${PV}"
+
+RDEPEND="
+	dev-libs/json-glib
+	net-libs/libnatpmp
+	net-libs/miniupnpc:=
+	clang? ( >=sys-devel/clang-6:= )"
```

Comment:
Are you sure about the subslot binder here? It feels... useless to me. Let me know what do you think, I can remove it later.

Reply:
Oh I see what you mean. I can take that out.

Reply:
I already merged this PR, I'll take it out before pushing. Thanks for checking!